### ISSUE TITLE
Setting up ability to start/stop timer on the apple watch

### ISSUE THREAD

**Issue by sohilpandya** [dwyl/time-apple-watch-app#12]
I've started working on the ability to start/stop a timer on the apple watch.

- [x] When a user selects a row, start and show the timer. 
- [x] When a user selects the row again, we need to stop the timer. 
- [x] get the total time in seconds for the period the timer was running
- [x] send a message to the iPhone when the user starts a timer on the apple watch (to set isRunning to true in the database)
- [x] send a message to the iPhone when the user stops the timer on the apple watch

Currently, it looks like you can only have one timer per view. which means that we cannot start multiple timers on the apple watch. So if a user has already started a timer and they select another task, this will have to stop the existing timer and start a new one for the given task. 
- [x] When a task is running, disable all the other rows. This means the user has to stop the task before starting another one. 🤔

**Comment by iteles** [comment]
> So if a user has already started a timer and they select another task, this will have to stop the existing timer and start a new one for the given task 

I don't see this as a problem, I think it's actually a feature.

Can we have the UX be that if I start a timer, any other running timers are stopped automatically? (i.e. 1 click instead of 2)

**Comment by sohilpandya** [comment]
At the end of last week, we were able fetch the list of projects on the Apple Watch, the list was also updated when a user deleted or added a new project. 

Over the last couple of days, I have added the following feature:
- Ability to start/stop a timer on the watch
- When a user selects start, we send a message to the phone which will save the information to core data
- When a user selects stop, we send a message to the phone which will update the information and add the total time for the given item in core data


Implementing the above feature has led to one of the watchConnectivity features (updating the list of projects on the apple watch when a new one is added) to break. I think I know why such an issue is happening, I need to spend some time refactoring the way I've implemented the timer and sending of messages from the watch to the phone. Once I complete that, we should be in a good place. 👍
